src/api/game.js

```diff
@@ -59,20 +61,26 @@ export const fetchSchedule = async () => {
   }
 };
 
-export const fetchPlayers = async () => {
+export const postBetting = async (bettingData) => {
   try {
-    // TO DO: 실서버 연결시 주석 해제 및 testDay를 today로 교체
-    // const today = formatDate(new Date(), "yyyyMMdd");
-    const testDay = "20210418";
-    const res = await fetch(`${API_URL}/games/${testDay}/players`, {
+    const token = document.cookie
+      .split("; ")
+      .find((row) => row.startsWith("access_token"))
+      .split("=")[1];
+
+    const res = await fetch(`${API_URL}/games/${testDay}/betting`, {
+      method: "POST",
       headers: {
         "Content-Type": "application/json",
-      }
+        Authorization: `Bearer ${token}`,
+      },
+      body: JSON.stringify({
+        ...bettingData,
+        date: testDay,
+      }),
```

Comment:
```suggestion
      },
      credentials: "include",
      body: JSON.stringify({
        ...bettingData,
        date: testDay,
      }),
```

다른 도메인으로 쿠키의 정보를 보내려면 credentials: "include" 를 써줘야 해요! 그래야 서버가 토큰을 받을 수 있을 것 같습니다!

Reply:
우왕 역시 로그인의 심장이다 감사합니다!!